README.md

```diff
@@ -9,20 +9,24 @@ The purpose of the SPDX License List is to enable efficient and reliable identif
+Please note, a license not on the SPDX License List can be included in an SPDX document by using a 'LicenseRef-' and including the full license text as per the specification. 
+
+## When 
+* SPDX License List releases are done on a quarterly basis (more or less) at the end of January, April, July, and October. We schedule our License List releases one month behind the usual quarterly calendar cadence to accommodate the reality that many people contributing have other commitments at quarter-end.
+* See https://github.com/spdx/license-list-XML/blob/master/RELEASE-NOTES.md for a summary of each release
+* All PRs to be included as part of release must be merged 1 week prior to release date to allow time for actual release work
+* Any new issues raised within the month of the next release will likely be tagged for the following release, unless it is an easy-to-resolve issue
 
 # This Repository
 This repository contains the XML source and schema files used to generate the authoritative, supported SPDX list file formats, including the web pages you see at [spdx.org/licenses](https://spdx.org/licenses/) and other generated data formats found in the [SPDX license-list-data repository](https://github.com/spdx/license-list-data).
 
 # How to contribute/participate
 We welcome participants and contributions! The SPDX License List is maintained by the SPDX Legal Team. Work and discussion is primarily done via:
 * **mailing list**: Please introduce yourself and let us know a bit about your interest in SPDX! The mailing list is our traditional form of communication. Join the mailing list, see archive, and manage your subscription [via lists.spdx.org](https://lists.spdx.org/g/Spdx-legal). 
-* **bi-weekly calls**: used to discuss topics and issues that may be difficult to only discuss via email or via Github. Information on the bi-weekly calls and the working area for the SPDX Legal Team, can be found on the [SPDX legal team wiki](https://wiki.spdx.org/view/Legal_Team)
-* **this Github repo**: comments and issues and PRs related to specific changes to the files that comprise the SPDX License List, e.g., update a URL, recommend additional markup for matching purposes, or other such changes. 
-
-
+* **bi-weekly calls**: We use bi-weekly conference calls to make decisions on topics and issues that may be difficult to only discuss via email or via Github. Bi-weekly conference call info is sent prior to the calls via the mailing list. If you join the mailing list, you should get a recurring invite at the beginning of each calendar year. Meeting minutes for the calls are in the [SPDX meetings repo](https://github.com/spdx/meetings/tree/master/legal); historical meeting minutes can be found at http://wiki.spdx.org/
```

Comment:
I always wonder whether bi-weekly is every two weeks or twice a week.

Reply:
sigh, it can mean both... I just never think of it that way because... 10+ years of this meeting being every other Thursday. have updated...